lib/msal-browser/src/app/IPublicClientApplication.ts

```diff
@@ -0,0 +1,19 @@
+import { AuthenticationResult, AuthorizationUrlRequest } from "@azure/msal-common";
+import { SilentFlowRequest, EndSessionRequest, AccountInfo } from "../";
+
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+export interface IPublicClientApplication {
```

Comment:
Shouldn't the return types be here?

Reply:
They are: `functionName(params): ReturnType;`

Reply:
How did I miss that .... haha